stable/kong/README.md

```diff
@@ -71,6 +71,14 @@ and their default values.
 | proxy.servicePort              | TCP port on which the Kong Proxy Service is exposed                              | `8443`              |
 | proxy.containerPort            | TCP port on which the Kong app listens for Proxy traffic                         | `8443`              |
 | proxy.nodePort                 | Node port when service type is `NodePort`                                        |                     |
+| proxy.http.enabled             | Enables http on the proxy                                                        | false               |
+| proxy.http.servicePort         | Service port to use for http                                                     | 80                  |
+| proxy.http.containerPort       | Container port to use for http                                                   | 8000                |
+| proxy.http.nodePort            | Node port to use for http                                                        | 32080               |
+| proxy.tls.enabled              | Enables TLS on the proxy                                                         | true                |
+| proxy.tls.containerPort        | Container port to use for TLS                                                    | 8443                |
+| proxy.tls.servicePort          | Service port to use for TLS                                                      | 8443                |
```

Comment:
This should be 443 and 8443.

Reply:
Thanks.  Missed another one.  

Updated.